From 6fcab88cf9bb0429d961deeac7d4f9c67bb810ea Mon Sep 17 00:00:00 2001
From: Jason Zeng <jason.zeng@intel.com>
Date: Mon, 15 Aug 2022 16:56:25 +0800
Subject: [PATCH] Enable Intel SPR HFI support for default config
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 6fcab88cf9bb0429d961deeac7d4f9c67bb810ea
Modified-by-SEL: No


Intel inclusion
category: feature
bugzilla: https://gitee.com/openeuler/intel-kernel/issues/I5MJBR
CVE: NA

Intel-SIG: openeuler_defconfig: Enable configs for Intel SPR HFI

--------------------------------------------

Enable necessary kernel configs for Intel SPR HFI in openeuler_defconfig.

Signed-off-by: Jason Zeng <jason.zeng@intel.com>
Signed-off-by: Aichun Shi <aichun.shi@intel.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/configs/openeuler_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 7354d0032767..e880456e3b45 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -4078,7 +4078,7 @@ CONFIG_SENSORS_W83627EHF=m
 CONFIG_SENSORS_ACPI_POWER=m
 CONFIG_SENSORS_ATK0110=m
 CONFIG_THERMAL=y
-# CONFIG_THERMAL_NETLINK is not set
+CONFIG_THERMAL_NETLINK=y
 # CONFIG_THERMAL_STATISTICS is not set
 CONFIG_THERMAL_EMERGENCY_POWEROFF_DELAY_MS=0
 CONFIG_THERMAL_HWMON=y
@@ -4110,6 +4110,7 @@ CONFIG_PROC_THERMAL_MMIO_RAPL=y
 # end of ACPI INT340X thermal drivers
 
 CONFIG_INTEL_PCH_THERMAL=m
+CONFIG_INTEL_HFI_THERMAL=y
 # end of Intel thermal drivers
 
 # CONFIG_GENERIC_ADC_THERMAL is not set
-- 
2.33.0

